change/@ni-nimble-components-bb93d4a0-4af8-4624-9783-709125113d4f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you update [CSS Guidelines](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md) to establish a policy that components should always use the new enum rather than a literal value when setting `z-index`?